### DISCUSSION TITLE
I did not receive invitation from my organization

### DISCUSSION THREAD

**Discussion by maitsysraja** [community/community#83865] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, 

I converted the 124@test.com account into an organization. The owner is 434@test.com when I invited myself as a team member at 456@test.com, I did not get any notification to join. Can you please check it out and suggest it to me?

Raja K

**Comment by yash-pal1** [comment]
If you didn't get an invitation to join your organization on GitHub, first, check your email and see if the invitation is there, maybe in your spam or junk folder. Also, log in to your GitHub account and look for notifications. If you can't find anything, ask the person in your organization who manages GitHub to check if they sent the invitation and, if not, to resend it. You can also check a special page on GitHub called "Invitations" to see if there's an invitation waiting for you. If all else fails, make sure your GitHub username is correct, and ask the organization admin for a direct invitation link or check the organization settings. If you're still having trouble, consider reaching out to GitHub Support for help.

**Comment by maitsysraja** [comment]
Hi, 

I have attached a screenshot. It says pending. However, I did not find any connection in my login.

![image](https://github.com/community/community/assets/153885676/d52591ab-24ce-49aa-9f3d-0af96149b8ef)

**Reply by yash-pal1** [reply]
try to resend

**Reply by maitsysraja** [reply]
I had done that several times. I am sure it is broken. I was accepted as a collaborator before. after I migrated to an organization and sent a new invite as an organization member, notification is not working. Where do I ask for support help?

**Reply by yash-pal1** [reply]
when you convert your account to organization

**Reply by maitsysraja** [reply]
Hi, Can you please give me some direction where I can redirect this issue/bug?

**Comment by soul-xx** [comment]
.....

**Comment by azwarnaim** [comment]
this solve already?

**Reply by AlbertShown** [reply]
Nop.
